week-9/Homework/mandatory/1-practice/weatherApp.js

```diff
@@ -0,0 +1,58 @@
+window.addEventListener("load", ()=> {
+    let long;
+    let lat;
+    let temperatureDescription = document.querySelector(
+        ".temperature-description"
+    );
+    let temperatureDegree =  document.querySelector(".temperature-degree");
+    let locationTimezone = document.querySelector(".location-timezone");
+    let iconEl = document.querySelector(".icon");
+    let temperatureSection = document.querySelector(".temperature");
```

Comment:
Good job here - are there any parts of the code you need help with at the moment ?

Reply:
Or bits that you need explaining again